src/common/footer/Footer.module.css

```diff
@@ -0,0 +1,69 @@
+.koros {
+    width: 100%;
+    fill: var(--hds-ui-color-black-5);
+    height: 60px;
+}
+
+.footer {
+    width: 100%;
+    flex-shrink: 0;
+    background-color: var(--hds-ui-color-black-5);
+    padding: 0 10px 20px;
+}
+
+.content {
+    width: 1200px;
+    margin: 0 auto;
+    display: flex;
+    flex-direction: column;
+}
+
+@media (max-width: 1200px) {
+    .content {
+        width: auto;
+    }
+}
+
+.logo {
+    height: 42px;
+    margin-left: -20px;
+}
+
+
+hr {
+    border-top: 1px solid var(--hds-ui-color-black);
+    margin: 15px 0;
+    width: 100%;
+}
+
+.textContainer {
+    display: flex;
+    justify-content: space-between;
+    opacity: 0.8;
+}
+
+
+.feedback {
+    text-decoration: underline var(--hds-ui-color-black);
+    font-weight: bold;
+}
+
+.links {
+    color: var(--hds-ui-color-black);
+    font-weight: bold;
+}
+
+.links:visited,
+.feedback:visited {
+    color: var(--hds-ui-color-black);
+}
+
+@media (max-width: 600px) {
```

Comment:
Could you write the styles for `.textContainer` and `.feedback` in a mobile-first manner as well?

Reply:
Changed